.ci/component_descriptor

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
-COMPONENT_PREFIXES="eu.gcr.io/gardener-project/gardener/oidc-webhook-authenticator" \
+COMPONENT_PREFIXES="europe-docker.pkg.dev/gardener-project/releases/gardener/oidc-webhook-authenticator" \
```

Comment:
@vpnachev : in this particular case, I think it does not matter (however, I agree your suggested value would be preferrable on the grounds of being consistent with other repositories from github.com/gardener.

Let me explain what this environment variable controls (I had to reverse-engineer that from `component-cli`, btw - this is not logic I invented).

github.com/gardener/gardener's `hack/.ci/component_descriptor` script (called by this one) invokes another command (`component-cli --image-vector <path/to/images.yaml> --component-prefixes <prefixes>`). This command inspects `images.yaml` and will heuristically classify / translate entries, like so:

- "external images" (mostly images from quay.io/docker-hub, most of which copied to gardener's OCI registry below `/3rd/...` subpath). Those are added as "external resources" to the component-descriptor
- "local" images (images built from sources of _this_ repository; detected by checking `sourceRepository` for equality to current component-name (which is equal to repository-URL as of our pipeline-template). those are skipped, as they are already automatically added to component-descriptor from pipeline-template
- gardener-component-references: if image-registry/repository starts with any of the given COMPONENT_PREFIXES, such entries are considered as OCM Component-References (even though they actually point to OCI Images)

Because in this repository, there is no "component-reference entry" + current component has a special-handling anyhow, it will not make any difference, regardless how the COMPONENT_PREFIXES value is defined.

[this is the re-implemented heuristics](https://github.com/gardener/cc-utils/blob/master/bin/component-cli#L179-L252) in case you are interested in that detail :-)